rpcd/playbooks/roles/rpc_maas/templates/rsyslogd_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
+label: "rsyslogd_process_check--{{ ansible_hostname }}"
+disabled    : false
+period      : "{{ maas_check_period }}"
+timeout     : "{{ maas_check_timeout }}"
+details     :
+    file    : process_check.py
```

Comment:
The plugin won't run inside the MaaS venv this way. You'll have to invoke it using the wrapper script. i.e. https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/roles/rpc_maas/templates/cinder_api_local_check.yaml.j2#L7